inc/healthcheck/class-cli-command.php

```diff
@@ -7,6 +7,15 @@
 use WP_CLI_Command;
 
 class CLI_Command extends WP_CLI_Command {
+
+	/**
+	 * Run health checks.
+	 *
+	 * @param array $args
+	 * @param array $args_assoc
+	 *
+	 * @return void
```

Comment:
No need for the void return hinting. I believe these should also use `@synopsis` WP CLI documentation aswell than params